tox.ini

```diff
@@ -3,6 +3,7 @@
 envlist = {py26,py27,py32,py33,py34,py35}
 
 [testenv]
+extras = tests
 deps =
```

Comment:
`deps` should be dropped since they all come from `extras`!

Reply:
good call, fixed ✅